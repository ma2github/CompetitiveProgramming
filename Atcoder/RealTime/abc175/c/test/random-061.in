960966363776849 233082336657006 363280752558186
