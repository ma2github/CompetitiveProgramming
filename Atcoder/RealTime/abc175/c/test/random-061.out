234404858660477
